app/src/main/java/com/hms/referenceapp/photoapp/ui/profile/ProfileUiState.kt

```diff
@@ -0,0 +1,19 @@
+package com.hms.referenceapp.photoapp.ui.profile
+
+import com.hms.referenceapp.photoapp.data.model.UserProfileUiModel
+
+data class ProfileUiState(
+    val loading: Boolean,
+    val isUserSignOut: List<Boolean>,
```

Comment:
var name could be isUserSignedOut (optional)

Reply:
Done✔